qt/python/mantidqt/mantidqt/widgets/workspacedisplay/table/group_model.py

```diff
@@ -0,0 +1,172 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+# coding=utf-8
+#  This file is part of the mantidqt package.
+from mantid.api import WorkspaceGroup
+from mantidqt.widgets.workspacedisplay.table.error_column import ErrorColumn
+from mantidqt.widgets.workspacedisplay.table.marked_columns import MarkedColumns
+from mantidqt.widgets.workspacedisplay.table.model import TableWorkspaceColumnTypeMapping
+from collections import defaultdict
+
+
+class GroupTableWorkspaceDisplayModel:
```

Comment:
There's a lot of similar code here and in `TableWorkspaceDisplayModel`  - is there any way this could inherit or is that not possible? if so it could remove quite a lot of the boiler plate getters etc.